pom.xml

```diff
@@ -485,6 +485,79 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<artifactId>jdeb</artifactId>
+				<groupId>org.vafer</groupId>
+				<version>1.6</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>jdeb</goal>
+						</goals>
+						<configuration>
+							<deb>${project.build.directory}/runelite_${project.version}.deb</deb>
+							<dataSet>
+								<data>
+									<src>${project.build.directory}/${project.build.finalName}.jar</src>
+									<type>file</type>
+									<dst>/usr/share/runelite/RuneLite.jar</dst>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/runelite</src>
+									<type>file</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/usr/games/</prefix>
+										<filemode>755</filemode>
+									</mapper>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/runelite-no-hw-acc</src>
+									<type>file</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/usr/games/</prefix>
+										<filemode>755</filemode>
+									</mapper>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/RuneLite.desktop</src>
+									<type>file</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/usr/share/applications</prefix>
+									</mapper>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/RuneLiteNoHwAcc.desktop</src>
+									<type>file</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/usr/share/applications</prefix>
+									</mapper>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/runelite.png</src>
+									<type>file</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/usr/share/runelite</prefix>
+									</mapper>
+								</data>
+								<data>
+									<src>${basedir}/src/deb/copyright</src>
```

Comment:
Instead of having the copyright in `src/deb`, it might be time to add a LICENSE file in the root of the repository, and use that.